app/models/scored_schools_list.rb

```diff
@@ -17,6 +17,10 @@ def position(school)
     schools_with_positions.select {|_position, schools| schools.include?(school)}.first.first
   end
 
+  def top_three
+    with_points.schools_at(0, 3)
+  end
+
   def with_points
     self.class.new(@scored_schools.reject {|scored_school| scored_school.sum_points.nil? || scored_school.sum_points <= 0})
```

Comment:
perhaps this can be simplified?
```
  def with_points
    positive_scored_schools = @scored_schools.select { |scored_school| scored_school.sum_points&.positive? }
    self.class.new(positive_scored_schools)
  end
```

Reply:
maybe even as a private method
```
  def with_points
    self.class.new(positive_scored_schools)
  end

  ....

  private

  def positive_scored_schools
    @scored_schools.select { |scored_school| scored_school.sum_points&.positive? }
  end
```